domain-models-runtime/src/test/java/org/folio/rest/tools/utils/ValidationHelperTest.java

```diff
@@ -110,4 +107,16 @@ public void authTest(TestContext context) {
       async.complete();
     });
   }
+  @Test
+  public void dupMsgTest(TestContext context) {
+    Async async = context.async();
+    String msg = "duplicate _id value violates unique constraint :55835c7c-2885-44f4-96ac-f03525b8e608";
+    Throwable t = new PgException("duplicate key value violates unique constraint \"123456\"", null, "23505",
+        "Key (_id)=(55835c7c-2885-44f4-96ac-f03525b8e608) already exists.");
+    ValidationHelper.handleError(t, r -> {
+      String responseMsg=((Errors) r.result().getEntity()).getErrors().get(0).getMessage();
+      context.assertEquals(msg, responseMsg);
+      async.complete();
+    });
+  }
```

Comment:
Thx for providing that would test that fails if your changes were not added.

Reply:
Hi, suggested changes has been done.